### ISSUE TITLE
Noob

### ISSUE THREAD

**Issue by kennetpostigo** [sindresorhus/meow#28]
Hey,

I'm new to CLI's and how to use node to create them. I stumbled upon this repo and I read the features that `meow` provides and I really want to use it. Looking at the README.md I'm not exactly sure where to start or how to use `meow`. 

I tried the example and I get the correct output but I'm a bit confused as to how this would be used to provide messages based on input to the CLI and how to act on input to the CLI.

I'm a noob when it comes to this and if I asked a dumb question above I'm sorry. Just want to be pointed in a better direction to understand how this all works and  what role it plays when building a CLI.

🙂

**Comment by sindresorhus** [comment]
I've been planning to write a blog post about it, but I'm too busy at the moment. This project was really just to make it easier for me to write a lot of CLI apps. I haven't really done much to promote it or make it easy to user for beginners. I'll keep this issue open to remind me to improve the docs.

You could look at some of my CLI's to see how it's used in practise:
- https://github.com/sindresorhus/pkg-dir-cli/blob/master/cli.js
- https://github.com/sindresorhus/sparkly-cli/blob/master/cli.js
- https://github.com/sindresorhus/latest-version-cli/blob/master/cli.js

Basically, `cli.input` gives you all the CLI arguments as an array. `cli.flags` gives you the flags, like `--foo=bar`.

**Comment by kennetpostigo** [comment]
Thank you for the links, I will go through them now 👍

**Comment by SamVerschueren** [comment]
@sindresorhus we could create a guide in the [guides](https://github.com/sindresorhus/guides) repo on how to create CLI apps in Node (with meow)?

**Comment by sindresorhus** [comment]
Yup, or that.

**Comment by SamVerschueren** [comment]
Might give it a go one of the following days.

**Comment by corysimmons** [comment]
I was a little confused at first. Try this:

`
const cli = meow(`...`)
console.log(cli)
`

then `node my_cli.js foo bar -baz qux` to see what input options you have available to you.

It's really that simple.

Other than that, Meow just offers a bunch of helpers to clean stuff up and do some stuff automatically for you.

**Comment by kennetpostigo** [comment]
@sindresorhus are you still planning on writing the blog post?

**Comment by corysimmons** [comment]
Probably not! Sindres never gives back to the JS community!

**Comment by SamVerschueren** [comment]
Sarcasm doesn't work on the interwebz...
